airflow/utils/dag_processing.py

```diff
@@ -569,7 +569,7 @@ def __init__(self,
         # Map from file path to the processor
         self._processors: Dict[str, AbstractDagFileProcessorProcess] = {}
 
-        self._heartbeat_count = 0
+        self._no_run = 0
```

Comment:
```suggestion
        self._num_run = 0
```

Please? Seeing `_no_run` I expected it to be a boolean/flag variable.

Reply:
I renamed this variable.